packages/component-library/src/components/table-and-list/mt-data-table/mt-data-table.vue

```diff
@@ -453,23 +463,27 @@
                   v-if="!(disableSettingsTable && disableEdit && disableDelete)"
                   class="mt-data-table__table-context-button"
                 >
-                  <a v-if="!disableEdit" href="#" @click.prevent="$emit('open-details', data)">
-                    {{ $t("mt-data-table.contextButtons.edit") }}
-                  </a>
-                  <mt-context-button v-if="!(disableDelete && disableEdit)">
-                    <mt-context-menu-item
-                      v-if="!disableEdit"
-                      :label="$t('mt-data-table.contextButtons.edit')"
-                      @click="$emit('open-details', data)"
-                    />
-
-                    <mt-context-menu-item
-                      v-if="!disableDelete"
-                      type="critical"
-                      :label="$t('mt-data-table.contextButtons.delete')"
-                      @click="$emit('item-delete', data)"
-                    />
-                  </mt-context-button>
+                  <slot name="action-button" v-bind="{ data }">
+                    <a v-if="!disableEdit" href="#" @click.prevent="$emit('open-details', data)">
+                      {{ $t("mt-data-table.contextButtons.edit") }}
+                    </a>
+                  </slot>
+                  <slot name="context-button" v-bind="{ data }">
```

Comment:
Same here. This would allows to override the context buttons at a whole. I would prefer a property instead of a slot here. Something like this:
```js
additionalContextButtons: {
    type: Array as PropType<Array<{
       // Here the needed information about the buttons
    }>>,
    required: false,
   default: undefined
}
```

Then we can iterate over this property in the template and can also adjust the `v-if` check for the context button:
```
<mt-context-button v-if="!(disableDelete && disableEdit) || additionalContextButtons?.length > 0">
```

And then do a v-for for all context buttons in the property

Reply:
I updated, please check again